doc/admin-api.md

```diff
@@ -504,6 +504,9 @@ In addition to the basic complex equalization algorithm selection, APISIX's Upst
 |timeout|optional| Set the timeout for connection, sending and receiving messages. |
 |name     |optional|Identifies upstream names|
 |desc     |optional|upstream usage scenarios, and more.|
+|pass_host            |optional|`pass` pass the client request host, `node` not pass the client request host, using the upstream node host, `rewrite` rewrite host by the configured `upstream_host`.|
```

Comment:
> `pass` pass the client request host

that is the default value, we should write this to the doc.

Reply:
I have created an issue about this: https://github.com/apache/apisix/issues/2134

feel free to fix it ^_^